OneCommunityGlobal/HighestGoodNetworkApp
# Changhao - added unit test cases for resetpasswordpopup.jsx

# Description
new unit tests cover edge cases for restpasswordpopup.jsx

## How to test:
1. check into current branch
2. do `npm install` and `npm test ResetPasswordPopup.test.js` to run this PR locally
3. verify all 12 unit tests are passed

## Screenshot:
<img width="703" alt="Screenshot 2024-01-04 at 6 59 41 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/104778270/2291e635-1b55-4115-98ae-0d8112743a56">
